### PR TITLE
Direct icon to deck

### LINKED ISSUE #6995: Add ability to launch directly into reviews
I would like to be able for Anki to launch me directly into reviews when I open the app, so that I don't have to click on a specific deck. Ideally, this feature would allow me to specify what deck it auto-starts the reviews on. When using the "back" button, it would just take me to the list of all decks, as it works now when reviewing.

I'm envisioning something like this in the settings:
[ ] Immediately start reviews on launch (checkbox)
Default deck used for immediate reviews: < deck name > (select input with all decks you have in collection)

I've posted a bounty for this on https://www.bountysource.com/issues/92534801-add-ability-to-launch-directly-into-reviews

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
## Purpose / Description
I like the idea to get a link opening directly the reviewer, as suggested in #6995. So you can long click on a deck, and add a shortcut opening the reviewer directly to here.

## Fixes

Fixes #6995

## Approach
Uses https://developer.android.com/guide/topics/ui/shortcuts/creating-shortcuts#java

## How Has This Been Tested?

Open deck picker, long click on a deck, click on "Create shortcut", add it on android, open another deck, close ankidroid, click on the new icon, see that the correct deck is directly opened.

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] Strings resources: All strings added as resources are unique or contain a comment for seemingly duplicate strings to explain the difference between both resources

### CODE REVIEW

**Review by david-allison-1 — COMMENTED** [review]
How does a shortcut get removed?

**Comment by Arthur-Milchior** [comment]
As any other app on Android main window, you long press it and click on delete/drop it on delete

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java [hunk 1]
```diff
@@ -2549,6 +2554,22 @@ public void exportDeck(long did) {
         showDialogFragment(ExportDialog.newInstance(msg, did));
     }
 
+    public void createIcon(Context context) {
+        // This code should not be reachable with lower versions
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.java:2559]: Could you use `@RequiresApi` instead?

**Review by david-allison-1 — APPROVED** [review]
Looks good, thanks!

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
This isn't non-functional

**Comment by Arthur-Milchior** [comment]
I’m not used too functional change. This became a reflex

**Comment by david-allison-1** [comment]
```
  
  Errors found:
  
  /home/travis/build/ankidroid/Anki-Android/AnkiDroid/src/main/java/com/ichi2/anki/dialogs/DeckPickerContextMenu.java:150: Error: Call requires API level 26 (current min is 16): createIcon [NewApi]
                  ((DeckPicker) getActivity()).createIcon(getContext());
                                               ~~~~~~~~~~
```

**Comment by Arthur-Milchior** [comment]
Rhis is why I requested api O to execute this code. It seems tto work whenn it was in a if instead of in a @reUire

**Comment by david-allison-1** [comment]
Normally I'd say add the annotation to `CONTEXT_MENU_CREATE_SHORTCUT`, but that doesn't work well with switch statements. Let's leave it how it was

**Review by david-allison-1 — APPROVED** [review]

**Pull request merged**